src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/partials/PaymentBuilderTokensTotal/types.ts

```diff
@@ -0,0 +1,9 @@
+import { type Token } from '~types/graphql.ts';
+
+export interface PaymentBuilderTokenItem extends Token {
+  amount: string;
+}
```

Comment:
The domain term for those would be "payouts", I think it's worth updating it here to avoid confusion with the `Token` type (which is a property rather than a parent of a payout)

Reply:
Do you mean that I should replace `amount` with `payouts` or something else?